### DISCUSSION TITLE
YOLOv1

### DISCUSSION THREAD

**Discussion by ALEXCS-eg** [community/community#35746] [Other Feature Feedback, Questions, & Ideas]
Dear friend
in my course in AI i learn YOLO v1 ,so i have something i don't understand
in the first ,How does the YOLO algorithm work?
secondly,what is 1-stage algorithm (YOLO v1) do correctly
Detect boundaries and Classification object in the same time
or Detect boundaries first
or Classification object first

i hope i can find answer

**Comment by jfullstackdev** [comment]
you mean this ? 

https://medium.com/mlearning-ai/object-detection-explained-yolo-v1-fb4bcd3d87a1

that might help

**Reply by ALEXCS-eg** [reply]
Thanks for your attention 
I read this before , i don't find sure answer for my question, i need any documentation include the answer
